### ISSUE TITLE
comparison between polygon and official library polygon-api-client ?

### ISSUE THREAD

**Issue by timotheecour** [pssolanki111/polygon#13]
I don't see polygon-api-client mentioned anywhere in this repo, which seems to be the official API (https://github.com/polygon-io/client-python); what advantanges does this library offer over polygon-api-client?

**Comment by pssolanki111** [comment]
Long ago,
- the official wrapper was unmaintained and incomplete and had issues. I came across the issues as I was doing a project using this API
- I created this as a maintained wrapper on requests from some community members in the tda-api discord server
- Polygon team themselves liked it but for some reason they decided to not accept this as official wrapper 
- so instead they updated the old lib, taking some inspiration from this one.

as of now, both are similar in terms of usages. This one has some additional utility functions to perform bulk operations, multi-threading and asyncio enabled and other additional functionality which isn't in the official lib.

@timotheecour 

p.s sorry for the huge delay in response. I've been busy recently with work. Beginning to pick the up the pace again
